content/src/licenses/moving-company.md

```diff
@@ -7,14 +7,17 @@ previewType: renewal
 callToActionLink: https://www.njconsumeraffairs.gov/Pages/licensingboards.aspx
 callToActionText: " Renew My Public Movers and/or Warehousemen License"
 ---
-You must renew your license every year, within 30 days of the expiration date. 
+
+You must renew your license every year, within 30 days of the expiration date.
 
 ### Renewal Requirements (If You Already Have a Registered Account)
```

Comment:
[pebble] Does this need a `bodyHeading`?

Reply:
See above about license data.